workflow/cron/controller.go

```diff
@@ -229,25 +230,26 @@ func (cc *Controller) syncAll() {
 			continue
 		}
 
-		err = cc.syncCronWorkflow(cronWf, groupedWorkflows[cronWf.UID])
+		ctx := context.Background()
```

Comment:
Why are we using background here? I understand that the way `syncAll` gets called makes it so you can't pass parameters, but it would still be possible to pass it in using a higher-level function definition.

Maybe we should also add a `cc.ctx` or at least use a high-level def

Reply:
This one might just be a typo where I didn't see that `ctx` is declared in the function signature. I'll remove line 233 here